src/components/dashboard/main-content/course-enrollments/tests/CourseEnrollments.test.jsx

```diff
@@ -52,29 +71,51 @@ const transformedLicenseRequest = {
   notifications: [],
 };
 
+const assignmentsData = {
+  contentKey: 'test-contentKey',
+  contentTitle: 'test-title',
+  contentMetadata: {
+    endDate: '2018-08-18T05:00:00Z',
+    startDate: '2017-02-05T05:00:00Z',
+    courseType: 'test-course-type',
+    enrollByDate: '2017-02-05T05:00:00Z',
+    partners: [{ name: 'test-partner' }],
+  },
+  state: 'cancelled',
+};
+
 hooks.useCourseEnrollments.mockReturnValue({
   courseEnrollmentsByStatus: {
     inProgress: [inProgCourseRun],
     upcoming: [upcomingCourseRun],
     completed: [completedCourseRun],
+    assigned: [cancelledAssignedCourseRun],
     savedForLater: [savedForLaterCourseRun],
     requested: [transformedLicenseRequest],
   },
   updateCourseEnrollmentStatus: jest.fn(),
 });
-const initialUserSubsidyState = {};
+const initialUserSubsidyState = {
+  redeemableLearnerCreditPolicies: [
+    {
+      learnerContentAssignments: [assignmentsData],
+    },
+  ],
+};
 const renderEnrollmentsComponent = () => render(
-  <IntlProvider locale="en">
-    <AppContext.Provider value={{ enterpriseConfig }}>
-      <UserSubsidyContext.Provider value={initialUserSubsidyState}>
-        <SubsidyRequestsContext.Provider value={{ isLoading: false }}>
-          <CourseEnrollmentsContextProvider>
-            <CourseEnrollments />
-          </CourseEnrollmentsContextProvider>
-        </SubsidyRequestsContext.Provider>
-      </UserSubsidyContext.Provider>
-    </AppContext.Provider>
-  </IntlProvider>,
+  <BrowserRouter>
```

Comment:
[suggestion] If you use `renderWithRouter` from `@edx/frontend-enterprise-utils`, you'd likely be able to remove the boilerplate around `BrowserRouter`.